codalab/worker/worker_run_state.py

```diff
@@ -221,6 +224,7 @@ def mount_dependency(dependency, shared_file_system):
             )
             logger.error(message)
             logger.error(traceback.format_exc())
+            logger.info(f'{run_state.bundle.uuid} is changing state to {message}')
```

Comment:
The message is already being logged in the previous lines.  I don't think it's necessary.